src/main/java/com/github/seratch/jslack/api/methods/response/users/UsersIdentityResponse.java

```diff
@@ -17,10 +18,24 @@
     public static class User {
         private String name;
         private String id;
+        private String email;
+        @SerializedName("image_24")
```

Comment:
Yes - I tried it without it but it didn't work.  It would probably work if I just named the fields like "image_24" instead of "image24" but I wanted to be consistent with com.github.seratch.jslack.api.model.User.